src/commands/util/Exec.ts

```diff
@@ -30,7 +29,7 @@ export class Exec extends Command
 
     async run(int: CommandInteraction): Promise<void>
     {
-        const ephemeral = int.options.getBoolean("ephemeral");
+        const ephemeral = int.options.get("ephemeral")?.value as boolean | undefined ?? null;
```

Comment:
`?? null` is unnecessary but it works so who cares

Reply:
/shrug